.devcontainer/cuda12.5-conda/devcontainer.json

```diff
@@ -15,10 +15,10 @@
   ],
   "hostRequirements": {"gpu": "optional"},
   "features": {
-    "ghcr.io/rapidsai/devcontainers/features/rapids-build-utils:24.10": {}
+    "ghcr.io/lithomas1/devcontainers/features/rapids-build-utils:24.10": {}
   },
   "overrideFeatureInstallOrder": [
-    "ghcr.io/rapidsai/devcontainers/features/rapids-build-utils"
+    "ghcr.io/lithomas1/devcontainers/features/rapids-build-utils"
```

Comment:
Can you please upstream whatever changes you had to make into https://github.com/rapidsai/devcontainers and revert these?

Reply:
Tracked here: https://github.com/rapidsai/devcontainers/pull/375